### ISSUE TITLE
maklermodul-bundle 2.8.0 can not be installed in contao 4.9.34 due to "Class 'Contao\ArrayUtil' not found"

### ISSUE THREAD

**Issue by peerlight** [pdir/maklermodul-bundle#29]
## Bug description
Installation in contao 4.9.34 crashes with 
```
# Process terminated with exit code 1
# Result: General error
```

caused by 
```

CRITICAL  [console] Error thrown while running command "cache:warmup --env=prod --no-ansi". Message: "Class 'Contao\ArrayUtil' not found" ["exception" => Error { …},"command" => "cache:warmup --env=prod --no-ansi","message" => "Class 'Contao\ArrayUtil' not found"]
In config.php line 34:
                                                                
  Attempted to load class "ArrayUtil" from namespace "Contao".  
  Did you forget a "use" statement for another namespace?       
                                                                
cache:warmup [--no-optional-warmers] [-h|--help] [-q|--quiet] [-v|vv|vvv|--verbose] [-V|--version] [--ansi] [--no-ansi] [-n|--no-interaction] [-e|--env ENV] [--] <command>
Script Contao\ManagerBundle\Composer\ScriptHandler::initializeApplication handling the post-install-cmd event terminated with an exception
In ScriptHandler.php line 97:
                                                                               
  An error occurred while executing the "cache:warmup" command: 11:51:36 CRIT  
  ICAL  [console] Error thrown while running command "cache:warmup --env=prod  
   --no-ansi". Message: "Class 'Contao\ArrayUtil' not found" ["exception" =>   
  Error { …},"command" => "cache:warmup --env=prod --no-ansi","message" => "C  
  lass 'Contao\ArrayUtil' not found"]                                          
                                                                               
  In config.php line 34:                                                       
                                                                               
    Attempted to load class "ArrayUtil" from namespace "Contao".               
    Did you forget a "use" statement for another namespace?                    
                                                                               
                                                                               
  cache:warmup [--no-optional-warmers] [-h|--help] [-q|--quiet] [-v|vv|vvv|--  
  verbose] [-V|--version] [--ansi] [--no-ansi] [-n|--no-interaction] [-e|--en  
  v ENV] [--] <command>                                                        
                                                                               

```
## Environment
PHP 7.4
contao 4.9.34
pdir/maklermodul-bundle 2.7.9
pdir/maklermodul-sync-bundle 2.6.3

**Comment by peerlight** [comment]
see https://github.com/pdir/maklermodul-bundle/commit/fdf1fb1fbbab2fdaf8899be743037bf85af375bc which added `use Contao\ArrayUtil;` and line 34 with `ArrayUtil::arrayInsert($GLOBALS['BE_MOD'], 1, ['pdir' => []]);` to src/Resources/contao/config/config.php

**Comment by MDevster** [comment]
It uses Contao\ArrayUtil, which is only available in Contao 4.10 and higher. Will be fixed in 2.8.1 today ;)
